### ISSUE TITLE
Feature request: support longer assume role credentials

### ISSUE THREAD

**Issue by et304383** [aws/aws-cli#3307]
https://aws.amazon.com/blogs/security/enable-federated-api-access-to-your-aws-resources-for-up-to-12-hours-using-iam-roles/

I searched the opened and closed issues list since March 28 both here and on botocore and I can't find an existing feature request.

I use profiles like this:

```
[profile myprofile]
output = json
region = us-east-1
role_arn = arn:aws:iam::123456789012:role/MyRole
source_profile = identity
mfa_serial = arn:aws:iam::987654321098:mfa/my.mfa
```

I'd like that when I do this:
```
aws --profile myprofile <stuff>
```

after inputting my MFA the credentials are good for up to 12 hours.

**Comment by joguSD** [comment]
Marking as a feature request. It looks like this is possible but not exposed currently.

**Comment by kylekrall** [comment]
👍

**Comment by joguSD** [comment]
This was released in yesterday's release and is available as of [v1.16.73](https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst#11673). 

Related docs [here](https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#using-aws-iam-roles).
